Code/SPActions.ts

```diff
@@ -23,13 +23,21 @@ module CSREditor {
                 if (window["WPQ" + wpqId + "FormCtx"]) {
 
                     var ctx = window["WPQ" + wpqId + "FormCtx"];
+
+                    // add fields to context
+                    var fields = [];
+                    for (var f in ctx.FieldControlModes) {
```

Comment:
FieldControlModes contains some system fields that aren't really customizable. Those fields should be excluded: Attachments, Created, Modified, Author, Editor, _UIVersionString